webpack.static-viz.config.js

```diff
@@ -47,4 +49,7 @@ module.exports = {
       "metabase-lib": LIB_SRC_PATH,
     },
   },
+  optimization: {
+    minimize: !shouldDisableMinimization
```

Comment:
@alxnddr you could use `env.WEBPACK_WATCH` to disable minification in that case https://webpack.js.org/api/cli/#environment-options